.github/workflows/workflow.yml

```diff
@@ -69,16 +45,12 @@ jobs:
                   name: eslint
                   path: ./reports/junit
     storybook:
+        needs: [setup, linters]
```

Comment:
storybook and linters are unrelated tasks

Reply:
This is just to ensure no two jobs can run simultaneously to avoid the potential problem with the docker permission.